data-prepper-plugins/json-codec/src/main/java/org/opensearch/dataprepper/jsonCodec/JsonInputCodec.java

```diff
@@ -1,36 +1,30 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package org.opensearch.dataprepper.plugins.source.codec;
+package org.opensearch.dataprepper.jsonCodec;
```

Comment:
Please use either of the following package names:

```
org.opensearch.dataprepper.plugins.json.codec
```

or 

```
org.opensearch.dataprepper.plugins.codec.json
```

Reply:
I see two JsonInputCodec.java file in this PR. Please cleanup and re-submit.

Also, for json, we need the json parsing functionality in a library so that we can re-use it in other places (like future DLQ source code)

Reply:
What exactly do you mean in a library as the existing `JsonInputCodec` can be used anywhere within the `data-prepper-plugins` for json parsing functionality?

Do you mean to use json parsing functionality inside other data-prepper packages instead of only  `data-prepper-plugins`?

Reply:
@mahesh724 , we should not have upper-case package names in Java. I think you can change the package to one of the original suggestions which should resolve both my comment and @kkondaka 's comment.

And yes, we should have only one `JsonInputCodec`.

Reply:
Mahesh - Would'nt renaming the package as per David's comment would address this issue

Reply:
@ashoktelukuntla yes renaming package to `org.opensearch.dataprepper.plugins.codec.json` and removing one JsonInputCodec.java class would resolve the issue.